scripts/airdrop/src/client.rs

```diff
@@ -0,0 +1,10 @@
+use lazy_static::lazy_static;
+use std::sync::Mutex;
+
+lazy_static! {
+    static ref PRISMA_CLIENT: Mutex<PrismaClient> = Mutex::new(PrismaClient::new());
+}
+
+pub fn get_prisma_client() -> &'static PrismaClient {
+    PRISMA_CLIENT.lock().unwrap()
+}
```

Comment:
This is interesting. What is going on here? I think simplest would be to use sqlx and just write raw sql for this.